bin/git-obliterate

```diff
@@ -19,5 +19,5 @@ then
 else
     # don't quote $range so that we can forward multiple rev-list arguments
     git filter-branch -f --index-filter "git rm -r --cached ""$file"" --ignore-unmatch" \
-        --prune-empty --tag-name-filter cat -- $range
+        --prune-empty --tag-name-filter cat -- "$range"
```

Comment:
`range` may contain multiple options, let's add a `shellcheck disable`.

Reply:
I'm not an expert here but converted range to array so that multiple values would be quoted correctly in (theoretical) case they'd contain IFS or glob characters. LMKWYT? Could also just `shellcheck disable=SC2086`. Thanks for quick feedback on these few items 😎